server/apps/microservices/src/processors/video-transcode.processor.ts

```diff
@@ -43,12 +45,10 @@ export class VideoTranscodeProcessor {
   @Process({ name: JobName.VIDEO_CONVERSION, concurrency: 2 })
   async handleVideoConversion(job: Job<IAssetJob>) {
     const { asset } = job.data;
-    const basePath = APP_UPLOAD_LOCATION;
-    const encodedVideoPath = `${basePath}/${asset.ownerId}/encoded-video`;
 
-    if (!existsSync(encodedVideoPath)) {
-      mkdirSync(encodedVideoPath, { recursive: true });
-    }
+    const encodedVideoPath = join(APP_MEDIA_CACHE_LOCATION, asset.ownerId, 'encoded-video');
+
+    this.storageRepository.mkdirSync(encodedVideoPath);
 
     const savedEncodedPath = `${encodedVideoPath}/${asset.id}.mp4`;
```

Comment:
Should this use `join()` instead?

Reply:
Probably, yes :-). I've went ahead and fixed many of these, but obviously missed this one. I'll fix it.